Examples/Scripts/MaterialMapping/GeometryVisualisationAndMaterialHandling.py

```diff
@@ -0,0 +1,784 @@
+                steering_map[index_to_names[v_index]] = index_info(v_index+1)
+            steering_map[index_to_names[v_index]].layers_no_approach.append(coords[4])
+        v_index = v_index+1
+
+    v_index = 0
+    for elements in index_to_extends_layers_bounds_cylinders:
+        for coords in elements:
+            if v_index in interesting_volumes:
+                if index_to_names[v_index] not in steering_map:
+                    steering_map[index_to_names[v_index]] = index_info(v_index+1)
+                steering_map[index_to_names[v_index]].boundaries.append(coords[4])
+        v_index = v_index+1
+
+    v_index=0
+    for elements in index_to_extends_layers_bounds_discs:
+        for coords in elements:
+            if v_index in interesting_volumes:
+                if index_to_names[v_index] not in steering_map:
+                    steering_map[index_to_names[v_index]] = index_info(v_index+1)
+                steering_map[index_to_names[v_index]].boundaries.append(coords[3])
+        v_index = v_index+1
+
+    v_index=0
+    for elements in index_to_extends_layers_approach_cylinders:
+        for coords in elements:
+            if coords[5] not in steering_map[index_to_names[v_index]].layers_with_approach:
+                steering_map[index_to_names[v_index]].layers_with_approach.append(coords[5])
+            if coords[4] not in steering_map[index_to_names[v_index]].approaches:
+                steering_map[index_to_names[v_index]].approaches.append(coords[4])
+        v_index = v_index+1
+
+    v_index = 0
+    for elements in index_to_extends_layers_approach_discs:
+        for coords in elements:
+            if coords[4] not in steering_map[index_to_names[v_index]].layers_with_approach:
+                steering_map[index_to_names[v_index]].layers_with_approach.append(coords[4])
+            if coords[3] not in steering_map[index_to_names[v_index]].approaches:
+                steering_map[index_to_names[v_index]].approaches.append(coords[3])
+        v_index = v_index+1
+
+    if dump_steering:
+        output_map = { "SteeringField" : steering_map}
+        with open(steering_file, 'w', encoding='utf-8') as f:
+            json.dump(output_map, f, default=dumper, ensure_ascii=False, indent=4)
+
+    # Once you have all the data in hands, you make a few nice plots to show volumes and surfaces
+    if plot:
+        import matplotlib.pyplot as plt
+        plt.rcParams.update({'figure.max_open_warning': 0})
+        from matplotlib.pyplot import cm
+        from itertools import cycle
+        import numpy as np
+
+        color = cm.rainbow(np.linspace(0, 1, len(index_to_extends_layers_cylinders)))
+
+        is_in_legend = []
+
+        plt.figure(figsize=(20,10))
+        # Plot all layers (w/o approach layers)
+        v_index = 0
+        for elements in index_to_extends_layers_cylinders:
+            for coords in elements:
+                A, B = [coords[3], coords[1]], [coords[2], coords[1]]
```

Comment:
Does having A & B help to understand this better? Anyways, maybe a wrapping function with a clear name:
`x_values, y_values = fromCoordsToXY(coords)` could make it easier to follow.